src/main/resources/.env

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL = jdbc:postgresql://localhost:5432/libraryManagementSys
+DATABASE_USERNAME = postgres
+DATABASE_PASSWORD = postAdmin
```

Comment:
The purpose for using environment variables is to hide sensitive information. You have done but you shouldn't push the environment variables file to GitHub that defeats the purpose for hiding the information. 
To fix that add the .env file to gitignore.

Then in your README.md file you will specify the variables that are needed in the application.
```
Environment variables

DATABASE_URL 
DATABASE_USERNAME 
DATABASE_PASSWORD 
```